build/page-3

```diff
@@ -263,12 +272,27 @@
 
         <ul  class="ons-list ons-list--bare">                <li class="ons-list__item">
 
+<a href="/" class="ons-list__link">Home</a>                </li>
+                <li class="ons-list__item">
+
 <a href="/page-1" class="ons-list__link">Page 1</a>                </li>
                 <li class="ons-list__item">
 
 <a href="/page-2" class="ons-list__link">Page 2</a>                </li>
                 <li class="ons-list__item">
 
+<a href="/page-3" class="ons-list__link">Page 3</a>                </li>
+                <li class="ons-list__item">
+
+<a href="/mkdocs_page1" class="ons-list__link">MKdocs Local Markdown</a>                </li>
```

Comment:
I think the links are the wrong way around, i.e if I select the Python Docstring link it is rendering a markdown file, if I select External Markdown it shows a docstring